src/wallet/wallet2.h

```diff
@@ -827,6 +846,14 @@ namespace boost
       a & x.amount;
       a & x.addr;
     }
+    
+    template <class Archive>
+    inline void serialize(Archive& a, tools::wallet2::address_book_row& x, const boost::serialization::version_type ver)
+    {
+      a & x.m_address;
+      a & x.m_payment_id;
+      a & x.m_description;
+    }
   }
```

Comment:
You probably want to add a BOOST_CLASS_VERSION for this type as well, or any further changes will invariably forget about it and break backward compatibility.